cmd/infra/powervs/create.go

```diff
@@ -77,8 +77,13 @@ const (
 	nodePoolManagementCreds         = "node-management-creds"
 	ingressOperatorCreds            = "ingress-creds"
 	storageOperatorCreds            = "storage-creds"
+
+	// Retry limit
+	retryLimit = 5
 )
 
+var timeoutErrorKeywords = []string{"status 522", "status 524"}
```

Comment:
I see another variable in the line no.33, wondering if we can merge both of these create a new var group.

Reply:
Grouped all the vars